kafka/modules/kafka/base/scripts/kafka_connect_run.sh

```diff
@@ -37,7 +37,7 @@ fi
 
 # enabling Tracing agent (initializes Jaeger tracing) as Java agent
 if [ "$STRIMZI_TRACING" = "jaeger" ]; then
-    export KAFKA_OPTS="$KAFKA_OPTS -javaagent:$(ls $KAFKA_HOME/libs/tracing-agent.jar)=jaeger"
+    export KAFKA_OPTS="$KAFKA_OPTS -javaagent:$(ls $KAFKA_HOME/libs/tracing-agent*.jar)=jaeger"
```

Comment:
why this change and so the relationship with same change in Strimzi upstream?

Reply:
Not sure if the patch upstream will be accepted, and if it is if it will be cherry-picked for the next 0.14.0 RC release